tests/test_cmd_test.py

```diff
@@ -30,6 +34,16 @@ def test_startup_timeout(self):
             )
             self._check_exit_code(test_command, exit_code=1)
 
+    @skip_if_environ("PLANEMO_SKIP_GALAXY_TESTS")
+    def test_tool_in_directory(self):
+        """Test with (single) tool in directory."""
+        with self._isolate():
+            test_artifact = os.path.join(TEST_DATA_DIR, "tools", "ok_test_assert_command.xml")
+            tempdir = mkdtemp()
```

Comment:
Could use the `TemporaryDirectory()` context manager instead.

Reply:
Good idea, I'll do that